store/src/java-test/com/zimbra/cs/html/owasp/OwaspHtmlSanitizerTest.java

```diff
@@ -158,39 +159,39 @@ public void testBug46948() throws Exception {
     @Test
     public void testBug98215() throws Exception {
         String html = "<a href=\"vbscript:alert(parent.csrfToken)\">CLICK</a>";
-        String result = new OwaspHtmlSanitizer(html, true, null).sanitize();
+        String result = new OwaspHtmlSanitizer(html, true, null).sanitize(false);
```

Comment:
You should add test cases to make sure all these conditions pass with jtidy on as well.

Reply:
All these test cases are passing with `JTidy` enabled.